uber/templates/registration/history.html

```diff
@@ -25,6 +25,24 @@
 {% endfor %}
 </table>
 
+    <h2>Page View History for {{ attendee.full_name }} {% if c.AT_THE_CON %}({{ attendee.badge }}){% endif %}</h2>
```

Comment:
Since we now track pageviews in a separate table, it makes sense to display them separately. This also means that they won't show up in the FODC anymore. In my opinion, that's a good thing -- they were mostly cruft in day-to-day inspections. They're only there in case we need to audit data being viewed but not saved, which we haven't yet had to do.